standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/tools/schematool/TestSchemaToolForMetastore.java

```diff
@@ -370,6 +370,44 @@ public void testSchemaUpgrade() throws Exception {
     schemaTool.verifySchemaVersion();
   }
 
+  /**
+   * Test schema upgrade checking that the alpha version is called
+   */
+  @Test
+  public void testSchemaUpgradeFromAlphaVersion() throws Exception {
```

Comment:
Aren't tests under `DbInstallBase` sufficient to cover the upgrade? If they are not sufficient maybe it would make sense to add additional tests there so that we cover all supported backends.

Reply:
With this test I was able to run them in unit tests and check the results.

Reply:
FYI: https://issues.apache.org/jira/browse/HIVE-26021 regarding my last comment.